api/register/lists.ts

```diff
@@ -27,24 +27,44 @@ class CountriesList {
 	tags: ['languages-list'],
 })
 class LanguagesList {
-	@response({ status: 201 })
+	@response({ status: 200 })
 	successfulResponse(@body body: Languages) {}
 }
 
+@endpoint({
+	method: 'GET',
+	path: '/user-types/list',
+	tags: ['user-types-list'],
+})
+class UserTypesList {
+	@response({ status: 200 })
+	successfulResponse(@body body: ProfileTypeItem[]) {}
+}
```

Comment:
But change this too😅
```suggestion
@endpoint({
	method: 'GET',
	path: '/profile-types/list',
	tags: ['profile-types-list'],
})
class ProfileTypesList {
	@response({ status: 200 })
	successfulResponse(@body body: ProfileTypeItem[]) {}
}
```